umi/sumi/tests/test_crossbar.py

```diff
@@ -10,11 +10,10 @@
 from switchboard import UmiTxRx, random_umi_packet, delete_queue
 
 
-def umi_send(x, n, ports):
-    import os
+def umi_send(x, n, ports, seed):
 
-    random.seed(os.getpid())
-    np.random.seed(os.getpid())
+    random.seed(seed)
+    np.random.seed(seed)
```

Comment:
it might be nice to pick a random system and just use one. seems odd to be using two different ones.